Assets/Scripts/CaveV2/SpawnObjects/LevelObjectSpawner.cs

```diff
@@ -272,6 +272,10 @@ private bool SpawnObjectsAtTags(Transform parent, bool retryOnFailure)
                         var newGameObject =
                             GameObjectUtils.SafeInstantiate(spawnAtTagParameters.InstanceAsPrefab, spawnAtTagParameters.Prefab, parent);
                         newGameObject.transform.SetPositionAndRotation(spawnAt.position.Value, spawnAt.rotation.Value);
+                        var spawnedObjectCaveNodeData = newGameObject.GetComponent<SpawnedObjectCaveNodeData>();
```

Comment:
The enemy spawner also utilizes spawn points; it would be nice if your solution extended to enemies as well.

Reply:
sure, added and pushed

Reply:
cool